### PR TITLE
build: add install target for openmldb

### LINKED ISSUE #868: automatically copy the utility scripts to the bin dir
at the end of compilation, the utility scripts (https://github.com/4paradigm/OpenMLDB/tree/main/release/bin) should be automatically copied to the install dir (by default the `bin` dir)

### PR DESCRIPTION
**Pull request by aceforeverd**
resolve #868 

- default install to ${PROJECT_ROOT}/openmldb
- remove old install logical in hybridse
- remove old `package_openmldb.sh`

now the install directory is exactly what the final release artifact contains

### CODE REVIEW

**Review by lumianph — COMMENTED** [review]
change docs/en/compile.md
line 65 默认 => Default

**Comment by aceforeverd** [comment]
> change docs/en/compile.md line 65 默认 => Default

resolved

File: hybridse/CMakeLists.txt [hunk 1]
```diff
@@ -240,19 +240,3 @@ if (COVERAGE_ENABLE)
                 DEPENDENCIES ${test_list})
     endif ()
 endif ()
-
-if (CMAKE_INSTALL_PREFIX STREQUAL "")
```
Comment (aceforeverd) [inline hybridse/CMakeLists.txt:244]: Rm it in hybridse workflow as well

**Review by zhanghaohit — APPROVED** [review]
LGTM

**Review by lumianph — APPROVED** [review]
LGTM

**Comment by aceforeverd** [comment]
[compilation improvement](https://trello.com/c/VcpqXsbY/3-compilation-improvement)

**Comment by lumianph** [comment]
[bugfix tracking](https://trello.com/c/wEQ9HW0k/25-bugfix-tracking)

**Pull request merged**